src/pages/term/nextBackButtons/nextBackButtons.module.sass

```
@@ -0,0 +1,4 @@
+.button-group
+    display: flex
+    gap: 2em
+    float: left
```

Should use logical directions, no physical ones.
`float` isn't recommanded to use. Use `justify-content` instead.
```suggestion
    justify-content: end
```